set-based-validation/src/main/java/io/axoniq/dev/samples/api/EmailAddressChangedEvent.java

```diff
@@ -0,0 +1,44 @@
+package io.axoniq.dev.samples.api;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public class EmailAddressChangedEvent {
+    private UUID accountId;
+    private String emailAddress;
+
+    public EmailAddressChangedEvent(UUID accountId, String emailAddress) {
```

Comment:
Why does this event not have any `@JsonProperty` annotations, like for example the `AccountCreatedEvent`? Or has the `Serializer` not been configured to use Jackson? If not, we can drop any reference of the `@JsonProperty` I think.